sa/precertificates.go

```diff
@@ -66,40 +66,31 @@ func (ssa *SQLStorageAuthority) AddPrecertificate(ctx context.Context, req *sapb
 			return nil, err
 		}
 
-		// With feature.StoreIssuerInfo we've added a new field to certStatusModel
-		// so when we try and use dbMap.Insert it will always try to insert that field.
-		// That will break when the relevant migration hasn't been applied so we need
-		// to use an explicit INSERT statement that we can manipulate to include the
-		// field only when the feature is enabled (and as such the migration has been
-		// applied).
-		csFields := certStatusFields
-		if features.Enabled(features.StoreIssuerInfo) && req.IssuerID != nil {
-			csFields += ", issuerID"
+		fieldNames := []string{}
+		for _, fieldName := range certStatusFields() {
```

Comment:
We call this twice, in this method, can we store the result here so we only need to call it once.